docs/userGuide/tipsAndTricks.md

```diff
@@ -34,18 +34,33 @@ For Markdown syntax: To display a literal character that are normally used for M
 
 ##### :fas-lightbulb: Using {% raw %}{% endraw %} to display `{{ content }}`
 
+By default, MarkBind processes any code in the form of `{{ content }}`. This is because Nunjucks (which is [supported by MarkBind](https://markbind.org/userGuide/markBindSyntaxOverview.html#support-for-nunjucks)) uses this syntax to [evaluate an expression, variable, or function calls](https://mozilla.github.io/nunjucks/templating.html#variables). For instance:
 
-To display the raw variable interpolation syntax using `{% raw %}{% endraw %}`, you would also need to add
-the `v-pre` attribute using markdown-it-attrs or as a HTML attribute.
+```
+{{ username }}
+```
+
+This will not display `{{ username }}` as a raw string, but instead, look up the variable `username` from the context and display its value.
+
+In general, to use this syntax as a raw string in a code block or a template, there are [two methods](https://jinja.palletsprojects.com/en/3.0.x/templates/#escaping) available. The syntax can either be encased as a variable like `{{ '{{' }}` and `{{ '}}' }}`, or alternatively, the entire code block can be encased with the raw-endraw tags: `{% raw %} {{ content }} {% endraw %}`. 
 
 <box type="info">
 
-This isn't necessary for `<code>` elements, markdown code fences and inline code though, which MarkBind automatically
-adds `v-pre` for.
+If using raw-endraw tags on html content, you would also need to add the `v-pre` attribute as an HTML attribute, like this:
 
-However, this does not change the need for `{% raw %}{% endraw %}`. Meaning, you can still use variables within your code!
-</box>
+```
+{% raw %}
+
+<p v-pre>{{ content }}</p>
 
+{% endraw %}
+```
+
+Otherwise, the content will not be displayed. 
```

Comment:
What does "the content" mean here?

Reply:
Ah, the "content" refers to the HTML content that are enclosed in double curly braces. I've fixed the wording in [f5a6772](https://github.com/MarkBind/markbind/pull/2118/commits/f5a6772a019b07a60cc8ea4a83033d3fcac37f5a)!